### ISSUE TITLE
Bug: cannot retrieve version information for pom outside work directory

### ISSUE THREAD

**Issue by mattdibi** [JActions/maven-version#1]
Hi there,
as per title we found an issue with this action when used with a repo with a `pom.xml` outside the work directory.
It reports:
```
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:3.0.0:exec (default-cli): Goal requires a project to execute but there is no POM in this directory (/home/runner/work/repo/repo). Please verify you invoked Maven from the correct directory. -> [Help 1]
```

even though the `pom.xml` path was specified.

There seems to be a bug in the action sources as the `pom.xml` path [is set](https://github.com/JActions/maven-version/blob/497a8ea3a19d17d429f34fefe7ad432dd05c48d4/action.yaml#L9) but [not used.](https://github.com/JActions/maven-version/blob/497a8ea3a19d17d429f34fefe7ad432dd05c48d4/action.yaml#L24).

I think the proper fix would be something like:

```yaml
runs:
  using: composite
  steps:
    - id: resolve-version
      shell: bash
      run: "echo \"::set-output name=resolved-version::\
        $(mvn -Dexec.executable=echo -Dexec.args='${project.version}' -f ${{inputs.pom}} --non-recursive exec:exec --quiet)\""
```

**Comment by JarvisCraft** [comment]
Thanks for reaching out,
For some reason I've forgotten to use this input parameter :D

I've commited a fix at [`GH-1`](https://github.com/JActions/maven-version/tree/GH-1) branch.

Could you please check it and tell if it resolves your issue?

**Comment by mattdibi** [comment]
Hi @JarvisCraft,

thanks for the quick feedback.

Looks like it's working:

![Screenshot 2022-06-29 at 12 02 00](https://user-images.githubusercontent.com/22748355/176410083-57c55f32-f6dd-4f73-9be7-9e40357b29e3.png)

This is what I was getting before:

![Screenshot 2022-06-29 at 12 02 24](https://user-images.githubusercontent.com/22748355/176410154-5295e9bb-523a-4d28-bb93-c40ac7766ee6.png)

I just changed the action reference:
<img width="749" alt="Screenshot 2022-06-29 at 12 04 33" src="https://user-images.githubusercontent.com/22748355/176410535-f047e3a4-89e9-459f-a1f6-972876a28fd2.png">

**Comment by JarvisCraft** [comment]
I've released the fix under `v1.0.1` tag
